Core/src/test/java/put/io/black/java/core/logic/FileManagerTest.java

```diff
@@ -0,0 +1,89 @@
+package put.io.black.java.core.logic;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.*;
+
+public class FileManagerTest {
+
+    private FileManager fileManager;
+    private int fileIterator = 0;
+
+    private int getFileIterator() {
+        fileIterator++;
+        return fileIterator;
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        fileManager = new FileManager();
+    }
+
+    @Test
+    public void fileManagerIsReadyTest() {
+        assertTrue(fileManager.isReady());
+    }
+
+    @Test
+    public void fileManagerSaveTest() {
+        String title = "title" + getFileIterator();
+        String text = "test";
+        assertTrue(fileManager.isReady());
+        fileManager.saveScenarioText(title, text);
+        assertTrue(new File(fileManager.PATH + title + ".txt").exists());
+    }
+
+    @Test
+    public void fileExistThenNotOverwrite() {
+        String text = "text";
+        String title = "title" + getFileIterator();
+        String title2 = "title" + (getFileIterator() - 1);
+        assertTrue(fileManager.isReady());
+        assertTrue(fileManager.saveScenarioText(title, text));
+        assertFalse(fileManager.saveScenarioText(title2, text));
+    }
+
+    @Test
+    public void readExistFile(){
+        String title = "title"+getFileIterator();
```

Comment:
Space between `"title"`, `+` and `get...`

Reply:
Add